scripts/govcms-deploy

```diff
@@ -10,30 +10,34 @@ LAGOON_ENVIRONMENT_TYPE=${LAGOON_ENVIRONMENT_TYPE:-production}
 GOVCMS_DEPLOY_WORKFLOW_CONFIG=${GOVCMS_DEPLOY_WORKFLOW_CONFIG:-import}
 # Determine the content strategy `import` vs `retain`.
 GOVCMS_DEPLOY_WORKFLOW_CONTENT=${GOVCMS_DEPLOY_WORKFLOW_CONTENT:-retain}
-
-# Ensure tmp folder always exists.
-mkdir -p /app/web/sites/default/files/private/tmp/
+# The location of the application directory.
+APP="${APP:-$APP}"
 
 # Check for presence of config files.
 set +e # Prevent script failure when assigning 0.
-# shellcheck disable=SC2012
-config_count=$(ls -1 /app/config/default/*.yml 2>/dev/null | wc -l)
-# shellcheck disable=SC2012
-dev_config_count=$(ls -1 /app/config/dev/*.yml 2>/dev/null | wc -l)
+# shellcheck disable=SC2012,SC2086
```

Comment:
SC2086 requires to put the statement into quotes, but we want the expansion to happen